src/client/actions/MessageReactionRemoveAll.js

```diff
@@ -24,6 +27,7 @@ class MessageReactionRemoveAll extends Action {
  * Emitted whenever all reactions are removed from a cached message.
  * @event Client#messageReactionRemoveAll
  * @param {Message} message The message the reactions were removed from
+ * @param {Collection} reactions The message reactions that were removed.
```

Comment:
Mmh, maybe here should be specified that only cached reactions are passed (so it's not the data from the API)?